packages/adblocker-benchmarks/package.json

```diff
@@ -14,19 +14,18 @@
     "directory": "packages/adblocker-benchmarks"
   },
   "scripts": {
-    "download-requests": "make requests.json",
```

Comment:
This needs to run as `postinstall` in order to download the required data for benchmarks.